tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -58,8 +58,32 @@ def check_node_replacement_verification_steps(
         AssertionError: If the node replacement verification steps failed.
 
     """
-    new_osd_node_name = node.wait_for_new_osd_node(old_osd_node_names, timeout=1500)
-    assert new_osd_node_name, "New osd node not found"
+    min_osd_nodes = 3
+    ocs_nodes = node.get_ocs_nodes()
+
+    if len(old_osd_node_names) <= min_osd_nodes:
+        log.info(
+            "We have the minimum number of osd nodes in the cluster. "
+            "Wait for the new created worker node to appear in the osd nodes"
+        )
+        new_osd_node_name = node.wait_for_new_osd_node(old_osd_node_names, timeout=1500)
+        assert new_osd_node_name, "New osd node not found"
```

Comment:
please make the assert messages more elaborative. 
Maybe something like:
```
"New osd node not found after <operation> while waiting for <timeout>"
```

Reply:
Yes, I will change it.

Reply:
Fixed